app/Transformers/ScoreTransformer.php

```diff
@@ -102,6 +102,7 @@ public function transformSolo(MultiplayerScoreLink|ScoreModel|SoloScore $score)
         if ($score instanceof SoloScore) {
             $extraAttributes['ranked'] = $score->ranked;
             $extraAttributes['preserve'] = $score->preserve;
+            $extraAttributes['processed'] = $score->processHistory()->first() !== null;
```

Comment:
I'm a bit concerned about this being in the transformer here – is there potential that it adds an overhead (+1 query?) in places we don't want it to?

Reply:
Hmm well it probably does but I'm not super sure how to do better... It does kinda need to be in the transformer because we'll want to use this in client to mirror this logic too.

Reply:
`processHistory` definitely needs to be eager loaded in all the collections it's used in at the very least.

Reply:
Well this is where I probably expose myself for a fraud but... do you have any good ways of determining _where_ that should be added? It's a bit impossible to do a "find all usages" as I would with a more statically-typed language (unless I am using inadequate tools, which I might be, since my tools for php are just vscode right now).

I see `MULTIPLAYER_BASE_PRELOAD` in `ScoreTransformer`, and `USER_PROFILE_INCLUDES_PRELOAD` next to it, and that they get fed into `with()` in queries, but those appear to only be specific usage contexts too... I also see `$defaultIncludes` but I don't think that is going to help me here any is it?

Reply:
`$defaultIncludes` is just the transformer defaults